cmd/provision/README.md

```diff
@@ -0,0 +1,77 @@
+# Provisioning
+
+This command offers all required functionality to provision an host, to join a Kubernetes cluster.
```

Comment:
Please remove the `comma` it disturbs the balance of the world

Reply:
Balance of the world has been recreated via https://github.com/kubermatic/machine-controller/pull/435/commits/5e8b411aea858f81800c5d443e2654a649f880dd